build.sbt

```diff
@@ -4,15 +4,28 @@ version := "0.1"
 
 scalaVersion := "2.11.12"
 
+
```

Comment:
what is this empty line for? I suppose it was added unintentionally since it does not improve code formatting.

Please, pay attention to every line you add (green colored) and remove (red colored).
You can go through a Pull Request files after it was created to double check that you have published exactly what indented to.

Reply:
Fixed..